low-level/framework/sspl_ll_d

```diff
@@ -740,6 +738,24 @@ def refresh_iem_log_file(signal_number, frame):
     else:
         logger.info(f"IEMSensor is not in running, ignoring signal to refresh file")
 
+def generate_cov_report(signal_number, frame):
```

Comment:
why we are adding code in production version of sspl_ll_d? We maintained earlier same code in different script, so as to enable when needed in dev/test env, same approach we need.
Add this code with comment markers in dev/test version of sspl_ll_d with start/stop_coverag_report.sh

Reply:
addressed.